<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registWidget</class>
 <widget class="QWidget" name="registWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>486</width>
     <height>377</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="inputsLayout">
    <item>
     <layout class="QVBoxLayout" name="usr_idLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户ID（5-25个字符，只能是数字和字母，不能为空）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usr_id"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="pswdLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>密码（8-16个字符，仅能包含数字、字母、*、_）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pswd">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="pswd_againLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>重复密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pswd_again">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="typeLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>要注册的用户类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type">
        <item>
         <property name="text">
          <string>普通用户</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>商家</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="phone_numberLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>电话号码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phone_number"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="usr_nameLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>用户名（1-15个字符，禁止出现三种括号、逗号、句号、点和单双引号）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usr_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="store_nameLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>店铺名称（不超过35个字符，禁止出现违法内容）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="store_name"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>560</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonsLayout">
    <item>
     <widget class="QPushButton" name="confirmButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>registWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
